packages/editor/src/client/api/index.ts

```diff
@@ -1340,6 +1342,12 @@ export type Query = {
 };
 
 
+export type QueryRemotePeerProfileArgs = {
+  hostURL: Scalars['String'];
```

Comment:
what's the difference between using `Scalars['String'];` or simply `String`?

Reply:
We use https://www.graphql-code-generator.com to generate this file out of the `*.graphql` files. So you don't have to review it in the PR. I think scalars belongs to something graphql specific.